packages/core/src/render3/interfaces/injector.ts

```diff
@@ -14,6 +14,11 @@ import {assertDefined, assertEqual} from '../../util/assert';
 import {TDirectiveHostNode} from './node';
 import {LView, TData} from './view';
 
+/**
+ * Offset from 'injectorIndex' where:
+ * - `TViewData[injectorIndex + TNODE]` => `TNode` associated with the current injector.
+ * - `LView[injectorIndex + TNODE]` => index to the parent injector.
```

Comment:
Should this comment not be:

```suggestion
 * - `LView[injectorIndex + PARENT_INJECTOR]` => index to the parent injector.
```

And if they need to be the same constant perhaps below it should be:

```ts
export const TNODE = 8;
export const PARENT_INJECTOR = TNODE;
```